### ISSUE TITLE
Customization of the OAuth flow page rendering

### ISSUE THREAD

**Issue by esstat17** [seratch/slack-edge#26]
Kazuhiro, interesting project and keep it up.

are you open for Pull Request? 

Any flexibility on render page? 

slack-cloudflare-workers/node_modules/slack-edge/dist/oauth/oauth-page-renderer.js

It's hard coded. I am finding the other way to modifying it.

**Comment by seratch** [comment]
Hi @esstat17, thanks for using this library! Currently, you can customize the behavior by passing oauth.start to SlackApp constructor. I plan to add further options for only customizing the rendering part but for now, please go ahead with oauth.start.

**Comment by seratch** [comment]
Thanks for sharing this feedback! The options to customize the rendering logic are now available in v0.11.

**Comment by esstat17** [comment]
Great job over the weekend! that was fast and this works for me.
```js
new SlackOAuthApp({
// ...

oauth: {
  callback: async (args) => {
  const { oauthAccess }: OAuthCallbackArgs = args;
  console.log(oauthAccess); //  IDOR
  const { incoming_webhook } = oauthAccess;
  return new Response(JSON.stringify(incoming_webhook), {
    headers: { 'Content-Type': 'application/json' }
  });
}

``` 

I can now modify it without hardcoded HTML rendered even without utilizing the newly `callbackRenderer` 

I will address here if i found any issues. Thank you!
